IPyth.sol

```diff
@@ -7,8 +7,29 @@ import "./PythStructs.sol";
 /// @dev Please refer to the guidance at https://docs.pyth.network/consumers/best-practices for how to consume prices safely.
 /// @author Pyth Data Association
 interface IPyth {
-    /// @dev Emitted when price feed with `id` is updated.
-    event PriceUpdate(bytes32 indexed id, uint64 publishTime);
+    /// @dev Emitted when an update for price feed with `id` is processed successfully.
+    /// @param id The Pyth Price Feed ID.
+    /// @param stored It's true if the price update is more recent and stored.
+    /// @param chainId ID of the source chain that the batch price update containing this price.
```

Comment:
Do we have a list of chain ID's anywhere? If so, might be worth referencing here.

Reply:
It's wormhole chainId, I like to avoid mentioning wormhole. I'll add it if you think it's good.
https://docs.wormholenetwork.com/wormhole/contracts#core-bridge
it's here ^

Reply:
Personally I think we should link to it, otherwise it's confusing.